templates/data/opensearch/index.html

```diff
@@ -0,0 +1,336 @@
+{% extends "data/base_data.html" %} 
+
+{% block title %}Canonical Charmed OpenSearch{% endblock %} 
+
+{% block meta_description %} 
```

Comment:
Could you make this single line please? That's usually the convention for these templates